contracts/pcv/utils/PCVDepositWrapper.sol

```diff
@@ -0,0 +1,48 @@
+pragma solidity ^0.8.4;
+
+import "../IPCVDeposit.sol";
+
+/**
+  @notice a lightweight contract to wrap old PCV deposits to use the new interface 
+  @author Fei Protocol
+  When upgrading the PCVDeposit interface, there are many old contracts which do not support it.
+  The main use case for the new interface is to add read methods for the Collateralization Oracle.
+  Most PCVDeposits resistant balance method is simply returning the balance as a pass-through
+  If the PCVDeposit holds FEI it may be considered as protocol FEI
+
+  This wrapper can be used in the CR oracle which reduces the number of contract upgrades and reduces the complexity and risk of the upgrade
+*/
+contract PCVDepositWrapper {
```

Comment:
If this contract is wrapping something else to expose methods for the interface, why doesn't it implement the new interface?

Reply:
It only implements the read parts of the interface. If it needed to literally wrap everything then it would have to implement withdraw and be given PCVController access. I guess we could implement the interface and make it a no-op. What do you think?

Reply:
let's make it inherit the new interface that is not deposit() but that the CR oracle reads & that is both inherited by PCVDeposits and BondingCurves